### PR TITLE
style(lic-checker): change print of abs path to only rel path

### PR DESCRIPTION
**Pull request by danielRep**
## PR Description

Small style changes on the output of the `license_checker.py` script to prevent the output of the absolute path of the file being parsed, which can make it to confusing.

### Type of change

- **style**: changes that do not affect the meaning of the code (formatting, typos, naming, etc.)

## Checklist:

- [x] The changes generate no new warnings when building the project. If so, I have justified above.
- [x] I have run the CI checkers before submitting the PR to avoid unnecessary runs of the workflow.

### CODE REVIEW

File: license_check.py [hunk 1]
```diff
@@ -35,15 +36,15 @@ def check_copyright(filename, copyright_notice):
             if copyright_notice in line:
                 return True
 
-        eprint(f'No copyright notice found in {filename}')
+        eprint(f'Copyright not found in ...\t\t{filename}')
```
Comment (josecm) [inline license_check.py:39]: I don't get the need to change the phrasing and the need for  `...\t\t`

File: license_check.py [hunk 2]
```diff
@@ -105,7 +106,6 @@ def main():
     spdx_expr = args.license
     if not spdx_expr:
         spdx_expr = 'Apache-2.0'
-        eprint(f'No SPDX expression supplied. Using {spdx_expr}.')
```
Comment (josecm) [inline license_check.py:108]: This seems to be unrelated to what the PR/commit is proposing. Should be in a different commit at least.

File: license_check.py [hunk 3]
```diff
@@ -117,12 +117,14 @@ def main():
     copyright_notice = args.copyright
     if not copyright_notice:
         copyright_notice = "Copyright (c) Bao Project and Contributors. All rights reserved"
-        eprint(f'No copyright notice supplied. Using {copyright_notice}.')
```
Comment (josecm) [inline license_check.py:120]: Same thing as line 108.

File: license_check.py [hunk 4]
```diff
@@ -35,15 +36,15 @@ def check_copyright(filename, copyright_notice):
...
+        eprint(f'Copyright not found in ...\t\t{filename}')
         return False
 
 
 def check_license(filename, spdx_expr):
 
     """Check if 'filename' has a SPDX identifier complying with 'spdx_exr'"""
 
-    # TODO: improve regex to support multiple licenses (AND, OR) and execeptions (WITH)
+    # TODO: improve regex to support multiple licenses (AND, OR) and exceptions (WITH)
```
Comment (josecm) [inline license_check.py:47]: We can keep this, but note that this kind of thing should be in a separate commit.

**Review by josecm — APPROVED** [review]

File: license_check.py (hunk 1 above)
Comment (danielRep) [reply license_check.py:39]: When I perform the checker on the partitioner I got several errors. The format has very messy to understand, that 's why I  tried to reformat the text to be more identical to license output in number of characters. And the ´\t\t´ is to line up the file paths with the license print.

**Review by AfonsoSantos96 — APPROVED** [review]

File: license_check.py (hunk 3 above)
Comment (danielRep) [reply license_check.py:120]: Done.

File: license_check.py (hunk 1 above)
Comment (josecm) [reply license_check.py:39]: I tried it myself and do agree. I'd suggest even simpler:

```suggestion
        eprint(f'Copyright not found: \t\t{filename}')
```

File: license_check.py [hunk 5]
```diff
@@ -81,7 +82,7 @@ def check_license(filename, spdx_expr):
 
                 return True
 
-    eprint(f'License not found in {filename}')
+    eprint(f'License not found in ...\t\t{filename}')
```
Comment (josecm) [inline license_check.py:85]:
```suggestion
    eprint(f'License not found: \t\t{filename}')
```

**Review by josecm — DISMISSED** [review]

**Comment by josecm** [comment]
@AfonsoSantos96 @miguelafsilva5

File: license_check.py (hunk 5 above)
Comment (danielRep) [reply license_check.py]: Changed.

File: license_check.py (hunk 1 above)
Comment (danielRep) [reply license_check.py]: Changed.

**Review by josecm — APPROVED** [review]

**Review by miguelafsilva5 — DISMISSED** [review]

File: license_check.py (hunk 1 above)
Comment (miguelafsilva5) [reply license_check.py]: @josecm Kindly resolve this conversation if the changes attend to your comments. Thank you.

File: license_check.py (hunk 5 above)
Comment (miguelafsilva5) [reply license_check.py]: @josecm Kindly resolve this conversation if the changes attend to your comments. Thank you.

**Comment by josecm** [comment]
@miguelafsilva5 @AfonsoSantos96 can you please re-aprove the PR? I had to force push because I mistakenly used merge instead of rebase to update the branch.

**Review by josecm — APPROVED** [review]

**Review by miguelafsilva5 — APPROVED** [review]
